lua/treesitter-context.lua

```diff
@@ -803,6 +806,26 @@ function M.setup(options)
   end
 end
 
+function M.go_to_previous_context()
```

Comment:
On second though a better name will be just `go_to_context`, since this won't work if the cursor isn't inside of a context.

E.g.

```lua
function foo()
  ....
end

<cursor>  -- go_to_previous_context() won't go to foo